ui/whitelabels.tsx

```diff
@@ -68,9 +68,9 @@ const envs = {
           <strong>Bitte speichere die Datei auf deinem Rechner ab.</strong>
         </p>
         <p>
-          <p>
+          <div>
```

Comment:
nice catch. Is either of them needed anyway? Wouldn't also work if you drop the tag wrapping `<Warning />`?

Reply:
It makes a slight styling difference. 

Without the wrapper:
<img width="399" alt="Screen Shot 2020-12-13 at 9 05 58 AM" src="https://user-images.githubusercontent.com/15280263/102006599-900ebf00-3d22-11eb-8f3c-3a0390b6fe16.png">

With the wrapper:
<img width="426" alt="Screen Shot 2020-12-13 at 9 07 15 AM" src="https://user-images.githubusercontent.com/15280263/102006608-99982700-3d22-11eb-91eb-febf8bd54e8c.png">

Putting the wrapper around makes it look exactly how it was before with the `<p>` around. Let me know which one you prefer @salimhb